retrieval/manager.go

```diff
@@ -56,8 +56,6 @@ type ScrapeManager struct {
 
 // Run starts background processing to handle target updates and reload the scraping loops.
 func (m *ScrapeManager) Run(tsets <-chan map[string][]*targetgroup.Group) error {
-	level.Info(m.logger).Log("msg", "Starting scrape manager...")
```

Comment:
why was this removed?

Reply:
it is not useful at all and no other Managers has this. 

It is useful only for the tsdb  - starting - started